UnityProject/Assets/Scripts/Tilemaps/Tiles/SimpleTile.cs

```diff
@@ -7,5 +11,22 @@ public class SimpleTile : BasicTile
 		public Sprite sprite;
 
 		public override Sprite PreviewSprite => sprite;
+
+		public bool CanBeHighlightedThroughScanners = false;
+
+		public event Action OnDestroyed;
+		[HideInInspector] public List<GameObject> AssoicatedSpawnedObjects;
+
+		[ShowIf(nameof(CanBeHighlightedThroughScanners))]
+		public GameObject HighlightObject;
```

Comment:
I think a tooltip would be nice.

Reply:
You can't have both NaughtyAttributes and a unity attribute at the same time, it breaks it

Reply:
Sad.